src/DataTableAbstract.php

```diff
@@ -507,6 +507,19 @@ public function startsWithSearch($state = true)
         return $this;
     }
 
+    /**
+     * Set multi_term search config at runtime.
+     *
+     * @param bool $multiTerm
+     * @return $this
+     */
+    public function setMultiTerm(bool $multiTerm = true)
```

Comment:
Maybe we should remove the `bool` type hint just to be sure this does not break on lower versions?

Reply:
Added a commit without the type hint, so it won't break anything in lower versions. bool was included on php 7 as a type hint so maybe someone is using it on 5.x still :)